jplag/src/main/java/de/jplag/Languages.java

```diff
@@ -0,0 +1,39 @@
+package de.jplag;
+
+import java.nio.file.ProviderNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+
+public class Languages {
+    public static List<String> getAllDisplayNames() {
+        return providers().stream()
+                .map(LanguageProvider::getDisplayName)
+                .sorted()
+                .collect(Collectors.toList());
+    }
+
+    public static List<LanguageProvider> providers() {
+        List<LanguageProvider> services = new ArrayList<>();
+        var loader = ServiceLoader.load(LanguageProvider.class);
+        loader.forEach(services::add);
+        return services;
```

Comment:
maybe throw an exception/ log an error if a name is duplicated.

Reply:
Oh yes, good catch, now it is possible to have duplicated names